examples/gno.land/p/gov/executor/callback.gno

```diff
@@ -0,0 +1,51 @@
+package executor
+
+import (
+	"errors"
+	"std"
+)
+
+const daoPkgPath = "gno.land/r/gov/dao/v2"
```

Comment:
Should not have this anymore.

Let’s try to make the DAO wrapping NewCallbackExecutor and inject itself’s address so that the callback executor contains the daopkgpath.

Reply:
Wrapped:

908b949
and
[6b3b81c](https://github.com/gnolang/gno/pull/2581/commits/6b3b81c3ffefd79a5fa3e63653f5b8e81a55ba77)

Reply:
I really like this approach, because the govdao pkg path is dynamic, and not hardcoded